### PR TITLE
Data source/loadbalancer backend address pool/backend_ip_configurations to be computed

### PR DESCRIPTION
**Pull request by marioanton**
- Adding compute to attribute backend_ip_configurations
- Minor lint issues

### CODE REVIEW

**Comment by leewilson86** [comment]
@tombuildsstuff 

This PR is a fix for a bug my colleague found with the following PR which I raised a few days ago (https://github.com/terraform-providers/terraform-provider-azurerm/pull/4605).

It fixes this error:

```
Resource 'data.azurerm_lb_backend_address_pool.beap' does not have attribute 'backend_ip_configurations' for variable 'data.azurerm_lb_backend_address_pool.beap.backend_ip_configurations'
```

File: azurerm/data_source_loadbalancer_backend_address_pool.go [hunk 1]
```diff
@@ -33,6 +33,7 @@ func dataSourceArmLoadBalancerBackendAddressPool() *schema.Resource {
 			"backend_ip_configurations": {
 				Type:     schema.TypeList,
 				Optional: true,
```
Comment (mbfrahry) [inline azurerm/data_source_loadbalancer_backend_address_pool.go]: Any attribute that isn't used to get the resource should remove the `Required/Optional: true` tag in favor of `Computed: true`

Comment (mariobede) [reply azurerm/data_source_loadbalancer_backend_address_pool.go]: Thanks, will sort that out.

**Comment by mariobede** [comment]
Changes commited and tested on my side.

**Comment by marioanton** [comment]
Changes commited.

**Pull request merged**